doc.ddoc

```diff
@@ -58,6 +58,7 @@ $(DIVID mobile-top,
 $(BODY)
 $(FOOTER)
 $(SCRIPTLOAD https://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js)
+$(SCRIPT window.jQuery || document.write('<script src="js/jquery-1.7.2.min.js">\x3C/script>'))
```

Comment:
clever ,  very very very very clever !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
you win